go/0217-contains-duplicate.go

```diff
@@ -1,12 +1,10 @@
 func containsDuplicate(nums []int) bool {
-	nums_map := map[int]int{}
+	nums_map := map[int]bool{}
```

Comment:
```suggestion
	nums_map := map[int]struct{}{}
```
It may make sense to use the `struct{}` type here because it doesn’t occupy any memory as opposed to `bool`.